reframe/core/buildsystems.py

```diff
@@ -779,6 +782,89 @@ def generated_modules(self):
         return self._eb_modules
 
 
+class Spack(BuildSystem):
+    packages = fields.TypedField(typ.List[str])
+    environment = fields.TypedField(str, type(None))
+    upstreams = fields.TypedField(typ.List[str])
+
+    def __init__(self):
+        super().__init__()
+        self.packages = []
+        self.environment = None
+        self.upstreams = []
+        self._prefix_save = None
+
+    def _prepare_scope(self):
+        os.mkdir('rfm_scope')
```

Comment:
I would write `rfm_spack_scope`. If you accept the idea, do not forget to change the `rfm_scope` from everywhere you have it... :-)